tests/test_giantsteps_tempo.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from mirdata import giantsteps_tempo, utils
+from tests.test_utils import run_track_tests
+
+
+def test_track():
+    default_trackid = '113'
+    data_home = 'tests/resources/mir_datasets/GiantSteps_tempo'
+    track = giantsteps_tempo.Track(default_trackid, data_home=data_home)
+
+    expected_attributes = {
+        'audio_path': 'tests/resources/mir_datasets/GiantSteps_tempo/audio/28952.LOFI.mp3',
+        'annotation_v1_path': 'tests/resources/mir_datasets/GiantSteps_tempo/giantsteps-tempo-dataset'
+                              '-0b7d47ba8cae59d3535a02e3db69e2cf6d0af5bb/annotations/jams/28952.LOFI.jams',
+        'annotation_v2_path': 'tests/resources/mir_datasets/GiantSteps_tempo/giantsteps-tempo-dataset'
+                              '-0b7d47ba8cae59d3535a02e3db69e2cf6d0af5bb/annotations_v2/jams/28952.LOFI.jams',
+        'title': '28952',
+        'track_id': '113',
+    }
+
+    expected_property_types = {
+        'tempo': list,
+        'tempo_v2': list,
+        'genre': str
+    }
+
+    run_track_tests(track, expected_attributes, expected_property_types)
+
+    audio, sr = track.audio
+    assert sr == 22050, 'sample rate {} is not 22050'.format(sr)
+    print(audio.shape)
+    assert audio.shape == (2646720,), 'audio shape {} was not (5294592,)'.format(
```

Comment:
Are this numbers right? Should they match?

Reply:
you are right

Reply:
done